### PR TITLE
Add webpack minimum version in peerDependencies

### PR DESCRIPTION
**Pull request by afdev82**
I added the minimum webpack version in the peerDependencies section of package.json.
See also #1323

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,6 +99,6 @@
   },
   "peerDependencies": {
     "typescript": "*",
-    "webpack": "*"
+    "webpack": "^5.20.0"
```
Comment (johnnyreilly) [inline package.json:102]: Thanks!  Why specifically 5.20.0 as the minimum?

Comment (afdev82) [reply package.json:102]: Because it's the version in the `devDependencies` section, so I assume the plugin is tested against this specific version.

Comment (johnnyreilly) [reply package.json:102]: It is - but it's not expressing a minimum version for users. To my knowledge, anything 5+ should be fine

Comment (afdev82) [inline package.json:102]:
```suggestion
    "webpack": "^5.0.0"
```

**Comment by johnnyreilly** [comment]
Thanks! Should ship with v9.2.2

**Pull request merged**